js/views/userPageVw.js

```diff
@@ -1413,6 +1437,92 @@ module.exports = baseVw.extend({
 
   },
 
+  addModeratorClick: function(e){
+    var $targ = $(e.target).closest('.js-addmoderator');
+
+    $targ.addClass('loading');
+
+    var user = this.model.get('user');
+    var self = this;
+
+    var modList = {};
+    modList.moderators = user.moderators;
+    modList.moderators[modList.moderators.length] = this.userProfile.get('profile').guid;
+    user.moderators = modList.moderators;
+
+    saveToAPI('', user, user.serverUrl + "settings",
```

Comment:
If you remove the user reference, this like can be done like this:

saveToAPI('', this.model.get('user'), this.model.get('user').serverUrl + "settings",